app/views/layouts/internal.html.erb

```diff
@@ -40,7 +40,7 @@
 
   <div class="container mt-3">
     <% flash.each do |type, message| %>
-      <div class="alert alert-<%= type %>">
+      <div class="alert alert-<%= type == "notice" ? "success" : "danger" %>">
```

Comment:
This makes sure we get the right color here. Not really specific to the issue but the old one wasn't quite right.